java/settings.gradle

```diff
@@ -1,4 +1,4 @@
-include ':add-graphics-renderer',
+include ':add-graphics-renderer', ':integrated-mesh-layer',
```

Comment:
Android Studio isn't clever enough to alphabetise this stuff. Could you slot it in below please?